### DISCUSSION TITLE
Total stats of all torrents lost each time restart.

### DISCUSSION THREAD

**Discussion by lilws** [jesec/flood#161] [General]
I'm not sure where to ask this, but since this git is the most active recently, I want to ask is there any option that I missed or can flood save all torrents stats? I run rtorrent-flood in docker.

Each time turn off rtorrent, it lost all stats that recorded in the top left corner.

<img width="243" alt="Screen Shot 2021-01-19 at 13 36 41" src="https://user-images.githubusercontent.com/12478478/104997905-f6db7280-5a5c-11eb-99e2-d90f448869c8.png">

**Comment by jesec** [comment]
This is expected behavior. Flood fetches session stats from torrent client.

**Reply by lilws** [reply]
Thank you, so I should asking from rtorrent author to make this feature available to save this stats.
And one more thing, it seems like I can't find the option to update tracker in flood. Like right click and choose Update tracker thing.

**Reply by jesec** [reply]
Go to context menu settings and enable the feature.

**Reply by lilws** [reply]
It looks like I'm missing something here or it doesn't appear.

<img width="496" alt="Screen Shot 2021-01-19 at 14 48 18" src="https://user-images.githubusercontent.com/12478478/105004101-0f9c5600-5a66-11eb-9675-edda27b8bf0e.png">

**Reply by jesec** [reply]
“Set trackers”

**Reply by lilws** [reply]
Oh, I think we have missunderstand here. I mean by "Update tracker" is sending a request to update peers/seeds from the tracker. Not *edit the tracker.

**Reply by jesec** [reply]
Flood doesn’t support manual tracker scrape yet.

**Reply by lilws** [reply]
Thanks, I hope you will add this feature in future release.
